packages/runner/src/errors.ts

```diff
@@ -9,3 +9,8 @@ export class PendingError extends Error {
     this.taskId = task.id
   }
 }
+
+export class AbortError extends Error {
+  name = 'AbortError'
+  code = 20
```

Comment:
What's the `20` for?

Reply:
The code for abort error in DOMException: https://developer.mozilla.org/en-US/docs/Web/API/DOMException/code